### DISCUSSION TITLE
Parent groups of GR-Company-BuyerExclusive and GR-Company-EconomicOperatorExclusive

### DISCUSSION THREAD

**Discussion by JanEmde** [OP-TED/eForms-SDK#859] [Q&A] [answered]
Why are GR-Company-BuyerExclusive and GR-Company-EconomicOperatorExclusive under the GR-Organisation-Section, although the associated fields can only be specified for buyers or tenderer? 

Wouldn't it make more sense to put GR-Company-BuyerExclusive under GR-Buyer and GR-Company-EconomicOperatorExclusive under GR-TenderingParty?

**Comment by YvesJo** [comment, marked as answer]
Hi,
This would lead to a possible repetition of information (everywhere the same organisation will appear) with the risk that there could be some discrepancy.
